src/process/process.cc

```diff
@@ -155,14 +163,32 @@ namespace blind
             all_cones, labels, stats, centroids);
 
         const cv::Point base(img.cols / 2, img.rows);
-        const auto mid = getDirection(nb_labels, centroids, yellow_cones, stats);
+        const auto mid =
+            getDirection(nb_labels, centroids, yellow_cones, stats);
+
+        cv::arrowedLine(original, base, mid, cv::Scalar(255, 255, 255));
+
+        return original;
+    }
+
+    int get_angle(cv::Mat &img)
+    {
+        cvtColor(img, img, cv::COLOR_BGR2HSV);
```

Comment:
Isnt this already computed in a previous step ?

Reply:
No as those are two separate function, but we could factorize

Reply:
Osef au pire